shared/lib/themes/helpers.js

```diff
@@ -9,6 +9,13 @@ var analytics = require( 'analytics' ),
 var ThemesHelpers = {
 	oldShowcaseUrl: '//wordpress.com/themes/',
 
+	getSignupUrl( theme ) {
+		// TODO:
+		// * Possibly add a `source` parameter
+		// * Can we point to Calypso NUX (/start) and pass it the theme id?
+		return '//signup.wordpress.com/signup/?source=calypso_showcase&theme=' + theme.id;
```

Comment:
From what I recall: as Hyperion we have "build the Themes funnel" as a milestone later, so for now we need to keep this "old" URL. Once it's ready, we can swap to the new proper funnel. :)